algosdk/v2client/indexer.py

```diff
@@ -145,7 +145,7 @@ def accounts(
             query["limit"] = limit
         if next_page:
             query["next"] = next_page
-        if min_balance:
+        if not min_balance is None:
```

Comment:
nit: I have a slight preference for the below due to readability

```suggestion
        if min_balance is not None:
```